spec/index.html

```diff
@@ -263,34 +338,43 @@
         };
         </pre>
         <p>
+          Initially all AccessibleNode properties must return null.
+        </p>
+        <p>
           All AccessibleNode properties, on setting, store the provided value.
-          On getting, they return the user-provided value if present, and otherwise
-          the value from the associated DOM Node's equivalent ARIA attribute, if any,
-          and otherwise they return null.
+          On getting, they return the user-provided value if present and valid,
+          and null otherwise. The validity of a property valid is specified in
+          greater detail below.
         </p>
         <p>
-          Values set on a DOM Node's AccessibleNode become the primary source of
+          Values set on a DOM element's AccessibleNode become the primary source of
```

Comment:
Which part of the explainer are you referencing? I recall us agreeing any value defined in AOM would override any content attribute (AIRA or otherwise).